features/model_specs/transactional_examples.feature

```diff
@@ -104,3 +104,23 @@ Feature: Transactional examples
       """
     When I run `rspec spec/models/thing_spec.rb`
     Then the examples should all pass
+
+  Scenario: Run in transactions with namespaced fixture
```

Comment:
Specs aren’t probably an option. Might be the example_app_generator or snippets (eg https://github.com/rspec/rspec-rails/blob/main/snippets/use_active_record_false.rb)?

Reply:
@pirj I removed the changes to the feature file for now. I'll see if I can come up with a way to test this otherwise, but I'm not familiar with the project setup so any help would be appreciated 🙂